packages/auth/webpack.config.mjs

```diff
@@ -252,24 +251,7 @@ export default env => {
          * React, React Native and React Navigation should be provided here because there should be only one instance of these modules.
          * Their names are used to match requested modules in this compilation.
          */
-        shared: {
-          react: {
-            ...Repack.Federated.SHARED_REACT,
-            requiredVersion: '18.2.0',
-          },
-          'react-native': {
-            ...Repack.Federated.SHARED_REACT_NATIVE,
-            requiredVersion: '0.71.8',
-          },
-          'react-native-paper': {
-            singleton: true,
-            requiredVersion: '5.0.0-rc.10',
-          },
-          '@react-native-async-storage/async-storage': {
-            singleton: true,
-            requiredVersion: '1.17.11',
-          },
-        },
+        shared: getSharedDependencies({eager: false}),
```

Comment:
worth adding a comment on why not eager?

Reply:
I've added description to the SDK on when to use eager